### ISSUE TITLE
Negative tags is not working as expected

### ISSUE THREAD

**Issue by Alfa-Q** [Alfa-Q/python-nozomi#5]
When getting posts, the posts retrieved can contain some of the negative tags, when it is expected that the posts retrieved cannot contain any of the negative tags.

#### Example
`api.get_posts(["a"], ["b", "c"])` will retrieve posts that can still contain one of those negative tags (`"b"` or `"c"`), but will never contain both (`"b"` and `"c"`).

It would be nice for it to be more explicit or add functionality to dictate whether the tags provided are independent.

#### Possible Solution:
`api.get_posts(["a"], [("b", "c")]` 
To get all posts that contain "a" but cannot contain BOTH tags `"b"` AND `"c"`

`api.get_posts(["a"], [("a"), ("b")])`
To get all posts that contain "a" but cannot contain a tag `"b"` OR `"c"`

Basically group tags using a tuple.

**Comment by SolitarySpiral** [comment]
resolution with negative tags is [here ](https://github.com/SolitarySpiral/python-nozomi-3.0.1/tree/v3.0.1-betta)
